src/common/constants.h

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright (C) by Oleksandr Zolotov <alex@nextcloud.com>
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY
+ * or FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License
+ * for more details.
+ */
+
+#pragma once
+
+namespace OCC {
+namespace CommonConstants {
+    const qint64 e2EeTagSize = 16;
```

Comment:
but then why not put it in the Constants namespace? What would be the difference between CommonConstants and Constants?

Reply:
@FlexW Because any other `.cpp` file may contain a namespace named `Constants` that only apply to that file, but, if we include this header there, we will be getting `already defined` compilation error. `CommonConstants` is a better name because it emphasizes that this namespace can be accessed from anywhere if needed, hence, it's not just `Constants` for a specific unit, but, it's a header that we can include into any unit when needed.

Reply:
Do we have a cpp file that has a namespace named Constants? I think a cpp file should use the anonymous namespace if it wants to namespace something but ok. We also don't name the OCC namespace the CommonOCC namespace :)

Reply:
I guess I would go with `Constants` as the name of the namespace to be aligned with for example `utility.h` has the `Utility` namespace in it
that way the rule is easy to memorize